.gitignore

```diff
@@ -119,3 +119,5 @@ __tests__/e2e/junit.xml
 
 .idea/
 seccomp/build
+
+test-projects/
```

Comment:
not sure why we need this on .gitignore?

Reply:
this is to init test projects if you have any as part of the repo.